spec/fixtures/api/gdk-backend-check/groot

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+echo "I'm Groot!"
```

Comment:
why are we logging this? the test doesn't seem to validate if this has been printed

Reply:
This again is just for debugging purposes.